fontbe/src/metrics_and_limits.rs

```diff
@@ -27,23 +27,21 @@ pub fn create_metric_and_limit_work() -> Box<BeWork> {
 }
 
 #[derive(Debug, Default)]
-struct GlyphLimits {
+struct Limits {
```

Comment:
I find confusing that there's an unqualified "Limits" struct and also one called "GlyphLimits" in the same module... why are these limits just "Limits" whereas the other are "GlyphLimits"? aren't they all about glyphs?

Reply:
This one is global and that one is for one glyph. I will rename this one FontLimits in hopes of clarifying the distinction.